### PR TITLE
Create Specs.md

### PR DESCRIPTION
**Pull request by mohitindukuri**

### CODE REVIEW

**Comment by puneethkanna** [comment]
Change it to the post layout

File: _posts/2020-05-26-realmetv-specs.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Powering this TV is the quad-core 64-bit MediaTek chipset (featuring ARM Cortex A53 CPU, clocked at up to 1.1GHz) and Mali-470 MP3 GPU. It comes with 1GB of RAM and 8GB of ROM. The TV comes with a 24W quad-speaker(2 speakers and 2 tweeters) with Dolby Audio support for rich sound stage.
+
+**Ports and Connectivity**:
+Speaking about connectivity it comes with 2.4GHZ WIFI, Bluetooth 5.0, and more. The fact that this does not come with WIFI 5 standard is a bit of a letdown.
+
+![RealmeTV Ports]({{ site.baseurl}}/assets/images/realme/realmeTV-ports.jpg)
+
+This TV  comes with a decent port selection, 3x HDMI, 2x USB, 1x Ethernet, 1x SPDIF, 1x AV port, 1x antenna in port, and a 3.5mm audio jack. 
+
+**Realme Smart TV: Remote**
```
Comment (SriSatyaLokesh) [inline _posts/2020-05-26-realmetv-specs.md]: replace all these bold to ####

**Review by SriSatyaLokesh — CHANGES_REQUESTED** [review]
please chane heading formats

File: _posts/2020-05-26-realmetv-specs.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+The remote looks similar to another smart remote from XIAOMI and ONE PLUS. It comes with a power On/Off button, a mute button, a Circular button for navigation at the top.below the circular button is the back, home, and settings button. Below them are the quick buttons for Netflix, YouTube, and Prime Video. It also comes with assistant button for quick voice commands and a plus, minus buttons to the left and right side of it, to increase and decrease volume. It can be operated both via the infrared and Bluetooth.
+
+![RealmeTV Smart Remote]({{ site.baseurl}}/assets/images/realme/realmetv-remote.jpg)
+
+**Price and Availability:**
```
Comment (puneethkanna) [inline _posts/2020-05-26-realmetv-specs.md]: Change the bold headings to ####(h4) headings

File: _posts/2020-05-26-realmetv-specs.md (hunk 1 above)
Comment (mohitindukuri) [reply _posts/2020-05-26-realmetv-specs.md]: ok bro

**Pull request merged**